src/index.js

```diff
@@ -8,6 +8,7 @@ import RedBox from './components/RedBox';
 import View from './components/View';
 import Text from './components/Text';
 import TextStyles from './sharedStyles/TextStyles';
+import Symbol from './symbol';
```

Comment:
`Symbol` isn't a great name because it shadows the JS primitive. Could we choose something else?